WebServices/TodoWCF/Droid/MainActivity.cs

```diff
@@ -19,6 +19,12 @@ protected override void OnCreate(Bundle bundle)
             global::Xamarin.Forms.Forms.Init(this, bundle);
 
             App.Speech = new Speech();
+            
+            // NOTE: Android emulator cannot access localhost but must
+            // use a local proxy. You may need to change this if testing
+            // on a device.
+            App.SoapUrl = "http://10.0.2.2:49393/TodoService.svc";
```

Comment:
Can't this be specified in the shared code library? You should be able to, doing something like:

```
SoapUrl =  Device.RuntimePlatform == Device.Android ? "https://10.0.2.2:5001" : "https://localhost:5001";
```